InventoryBuilder.py

```diff
@@ -253,12 +258,18 @@ def create_nsxt_objects(self, target: str, token: str):
             nsxt_mgmt_plane.add_adapter(adapter)
 
         nsxt_mgmt_cluster = Vrops.get_nsxt_mgmt_cluster(vrops, target, token, [a.uuid for a in nsxt_adapter])
+        nsxt_mgmt_nodes = Vrops.get_nsxt_mgmt_nodes(vrops, target, token, [c.uuid for c in nsxt_mgmt_cluster])
 
         for adapter_object in nsxt_mgmt_plane.adapter:
             for mgmt_cluster in nsxt_mgmt_cluster:
                 if mgmt_cluster.parent == adapter_object.uuid:
                     adapter_object.add_mgmt_cluster(mgmt_cluster)
                     logger.debug(f'Collecting NSX-T management cluster: {mgmt_cluster.name}')
+            for mgmt_cluster_object in adapter_object.management_cluster:
```

Comment:
```suggestion
            for mgmt_cluster_object in nsxt_adapter.management_cluster:
```
This is already in code, but down in the get_nsxt_* functions you are using `nsxt_adapter` instead of `adapter_object`. Since it is the same I would find it easier to read with the exact same naming. Applies to all `adapter_object` occurences.

Reply:
Fair enough. Changed the naming for the object.